plugins/removeAttrs.js

```diff
@@ -81,12 +90,16 @@ exports.fn = function(item, params) {
             // prepare patterns
         var patterns = params.attrs.map(function(pattern) {
 
-                // apply to all elements if specifc element is omitted
+                // if no element separators (:), assume it's attribute name, and apply to all elements and regardless of value
```

Comment:
maybe should say "to all element regardless of value" (I just removed the extraneous 'and')

Reply:
Maybe but I was trying to emphasize that the plugin will make 2 assumptions if there are no separators, 1) match all elements and 2) match any value.

Reply:
Maybe:

>assume it's attribute name, and apply to all elements (regardless of value)

Reply:
Doesn't parentheses imply "lesser importance" (e.g. afterthought)? Which is not exactly the message intended to convey.

Thanks

Reply:
I suppose you could state something like:

> if no element separators (:), apply the pattern matching to attribute name while otherwise matching on all elements and values

Seems to express the regex your join will result in.

Reply:
Sorry but I don't understand how we got so affixed on some Grammer in a comment within the code. I think the intended message comes across pretty well in the original version (which for some reason - for me at least - wasn't so in any of the suggestions). Can we get past this please?

Thanks

Reply:
👍